example/App.js

```diff
@@ -9,9 +9,12 @@
  */
 
 import React, {Component} from 'react';
-import {StyleSheet, Text, View, Button} from 'react-native';
-import {SSHClient} from '@ridenui/react-native-riden-ssh';
+import {StyleSheet, Text, View, Button, NativeEventEmitter} from 'react-native';
+import {SSHClient} from '@ridenui/react-native-riden-ssh-test/build/dist/index';
 import {SSH_HOST, SSH_USER, SSH_PORT, SSH_PASSWORD} from '@env';
+import MessageQueue from 'react-native/Libraries/BatchedBridge/MessageQueue.js';
+// import BatchedBridge from 'react-native/Libraries/BatchedBridge/BatchedBridge.js';
+// import RCTDeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter.js';
```

Comment:
Can we remove those?

Reply:
No. This is just the example app and need this in the future for quick debugging